cmd/cmd.go

```diff
@@ -914,8 +908,25 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(cmd *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
+
+	client, err := api.ClientFromEnvironment()
+	if err != nil {
+		return
+	}
+
+	serverVersion, err := client.Version(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	fmt.Printf("server version %s\n", serverVersion)
```

Comment:
Let’s print this only if they defer, and show “host version” since the user is presumably using OLLAMA_HOST here.

Reply:
I went with only printing this when the server version differs from the client. If your assertion is correct, the result is the same.